scripts/linters/pylint_extensions.py

```diff
@@ -1612,7 +1612,7 @@ class DivisionOperatorChecker(checkers.BaseChecker):
     priority = -1
     msgs = {
         'C0015': (
-            'Division Operator is used.',
+            'Do not use division operator.',
             'division-operator-used',
             'Please use python_utils.divide() instead of the "/" operator'
```

Comment:
Which message is the user-facing? (Whether line 1615 or 1617) TO me 1617 looks much more clear.

Reply:
Swapped 1615 with 1617.